tests/StorageBackendTest.php

```diff
@@ -2,76 +2,36 @@
 
 namespace Keboola\ManageApiTest;
 
-use Keboola\ManageApi\Backend;
 use Keboola\ManageApi\ClientException;
+use Keboola\StorageApi\Workspaces;
 
 class StorageBackendTest extends ClientTestCase
 {
-    public function supportedNonDefaultBackends(): array
-    {
-        return [
-            [Backend::REDSHIFT],
-            [Backend::SYNAPSE],
-        ];
-    }
-
     /**
-     * @dataProvider supportedNonDefaultBackends
-     * @param string $backendName
+     * @dataProvider storageBackendOptionsProvider
      */
-    public function testStorageAssignBackend(string $backendName): void
+    public function testCreateStorageBackend(array $options)
     {
-        // get redshift and synapse backend
-        $backends = $this->client->listStorageBackend();
-        $backendToAssign = null;
-        foreach ($backends as $item) {
-            if ($item['backend'] === $backendName) {
-                $backendToAssign = $item;
-            }
-        }
-        if (!$backendToAssign) {
-            $this->fail(sprintf('%s backend not found', ucfirst($backendName)));
-        }
+        $maintainerName = self::TESTS_MAINTAINER_PREFIX . sprintf(' - test managing %s storage backend', $options['backend']);
+
+        $newBackend = $this->client->createStorageBackend($options);
+
+        $this->assertBackendExist($newBackend['id']);
+
```

Comment:
tady bys měl kontrolovat, že ten backend co ti přišel v options je fakt snowflake a ne třeba teradata nebo redshift (dole to pak automaticky plníš do defaultConnectionSnowflakeId